styles/globals.css

```diff
@@ -32,6 +32,7 @@ nav, footer {
 
 nav {
   border-bottom: 1px solid #333;
+  background-color: #6D165D;
```

Comment:
prefer to keep the background black.

Reply:
Sorry, I accidentally committed this unrelated change from when I was playing around with styling. Definitely not needed.